plotify/plotify.py

```diff
@@ -118,7 +141,11 @@ def _format_data(df, value, x, plot_by=None, color_by=None, aggregate=True):
         message = 'Number of colors per plot exceeds maximum, MAX_COLORS = ' + str(MAX_COLORS)
         raise Exception(message)
 
-    df_new = df.groupby([SUBPLOT_COLUMN_NAME, COLOR_BY_COLUMN_NAME, x])[value].sum().reset_index()
+    if isinstance(value, dict):
+        df_new = df.groupby([SUBPLOT_COLUMN_NAME, COLOR_BY_COLUMN_NAME, x])[[value['numerator'], value['denominator']]].sum().reset_index()
+        df_new[value['name']] = df_new[value['numerator']]/df_new[value['denominator']]
```

Comment:
make sure to import he __future_division thingggg #python2

Reply:
good point

Reply:
it's already there